flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -389,14 +389,41 @@ def check_valid_reference_velocity(self) -> Self:
     @property
     def mach(self) -> pd.PositiveFloat:
         """Computes Mach number."""
-        return self.velocity_magnitude / self.thermal_state.speed_of_sound
+        return (self.velocity_magnitude / self.thermal_state.speed_of_sound).value
 
     @pd.field_validator("alpha", "beta", "thermal_state", mode="after")
     @classmethod
     def _update_input_cache(cls, value, info: pd.ValidationInfo):
         setattr(info.data["private_attribute_input_cache"], info.field_name, value)
         return value
 
+    @pd.validate_call
+    def reynolds_number(self, length_unit: LengthType.Positive):
+        """
+        Computes the Reynolds number defined in Flow360 solver.
```

Comment:
```suggestion
        Computes the Reynolds number defined in Flow360 solver.
```Computes length_unit based Reynolds number